### ISSUE TITLE
is this kind of outdated?

### ISSUE THREAD

**Issue by Maxhodges** [warehouseman/meteor-swagger-client#1]
become Promises are part of Meteor since v1.3

**Comment by martinhbramwell** [comment]
Yes.  Meteor 1.3 rendered obsolete not only the work of this repo but, at the same time, it wrecked the tutorial I made  :  [Meteor-CI](http://martinhbramwell.github.io/Meteor-CI-Tutorial/)

**Comment by Maxhodges** [comment]
i see. I'm just getting started with swagger. could be cool to have a swagger codegen for Meteor!

**Comment by martinhbramwell** [comment]
I agree.  Unfortunately, I've been allowing myself to get sidetracked by too many 'cool' things recently.  I recognized too late that Meteor had let itself get carried away with tech churn.  I'm trying to focus on Mantra as a way to get a stable(ish) platform to grow from.  I have a [fork of Meteor Mantra Kickstarter](https://github.com/warehouseman/meteor-mantra-kickstarter) that I'm completely cleaning up and upgrading to the latest versions of everything: Meteor, React, Astronomy, etc.

Every "kickstarter" /  "boilerplate", blaa blaa blaa I've found is either partially done and abandoned or reasonably complete but completely out of date.
